arelle/FormulaEvaluator.py

```diff
@@ -697,6 +697,21 @@ def implicitFilter(xpCtx, vb, facts, uncoveredAspectFacts):
 
     return _facts
 
+def orderByAspects(xpCtx, facts, aspects):
+    for aspect in aspects:
+        facts = orderByAspect(xpCtx, facts, aspect)
```

Comment:
Is it really intended to reorder the whole list rather than using aspects as "tie break" (like SQL ORDER BY)?  e.g., for [[id:11,a:1,b:2], [id:22,a:2,b:1]] and aspects=[a,b], this appears to return [22,11].

Is it intended that `aspects` is a `set` or a `list`?  For tie-breaking, it seems like it should be a `list`, but it looks like `filteredFactsPartitions` calls `factsPartitions` with `aspectsCovered`, which seems to return a `set`.